python/scripts/rfi_receiver.py

```diff
@@ -787,6 +789,73 @@ def http_server2():
     httpd.serve_forever()
 
 
+def rfi_zeroing():
+    # Wait until the correct UTC time (deals with daylight savings time)
```

Comment:
Doesn't this need a loop, otherwise it'll only run once?

Reply:
Also, if I restart this script half way through the solar transit (after the transit itself, but before the end of the window), isn't it going to sleep until the next solar transit when it turns it off again.

It should presumably ensure that zeroing is off, and then wait until the end of transit window when it turns it back on.

Similarly if it starts up between transits, it should ensure that zeroing is turned on.

Reply:
How do I check RFI zeroing is turned on/off? Can I do something like: `r = requests.get(url, data=json.dumps(payload), headers=headers)`?

Reply:
Also, if the script starts up and zeroing is off am I to assume that it is going through a solar transit window?

Reply:
It's pretty cheap to send the update, and this script shouldn't restart often, so I would just *always* set the state to what it should be when things start up. i.e. if you start during the window the first thing you do is send the command to turn zeroing off, and then wait until the end of the period.

Reply:
Ah okay thanks, I will do that then.

Reply:
I've added that edge case now.